### PR TITLE
OSDOCS-6130: Added IMDS information to OSD

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
`enterprise-4.13+`

Issue:
[OSDOCS-6130](https://issues.redhat.com/browse/OSDOCS-6130)

Link to docs preview:

- [OSD cluster creation](https://62577--docspreview.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-an-aws-cluster#osd-create-aws-cluster-ccs_osd-creating-a-cluster-on-aws) with AWS
image

QE review:
- [x] QE has approved this change.

Additional information:
This PR adds the IMDS content to OSD from ROSA. This PR should remain as a work-in-progress until [OSDOCS-6908](https://issues.redhat.com/browse/OSDOCS-6908) has been merged to determine the correct location for this information.

### CODE REVIEW

File: modules/osd-create-cluster-ccs.adoc [hunk 1]
```diff
@@ -166,6 +166,18 @@ PVs created by using any other storage class are still encrypted, but the PVs ar
 After your cluster is created, you can change the number of compute nodes in your cluster, but you cannot change the compute node instance type in a machine pool. The number and types of nodes available to you depend on your {product-title} subscription.
 ====
 
+. Choose your preference for the Instance Metadata Service (IMDS) type, either using both IMDSv1 and IMDSv2 types or requiring your EC2 instances use only IMDSv2. The two ways to access instance metadata from a running instance are:
```
Comment (jneczypor) [inline modules/osd-create-cluster-ccs.adoc:169]: I believe a "to" needs to be added after "EC2 instances".

**Review by skopacz1 — APPROVED** [review]
LGTM! I like how clean and easy to read this all is. Please wait until @adellape approves my review before proceeding, thanks!

File: modules/osd-create-cluster-ccs.adoc [hunk 2]
```diff
@@ -166,6 +166,23 @@ PVs created by using any other storage class are still encrypted, but the PVs ar
 After your cluster is created, you can change the number of compute nodes in your cluster, but you cannot change the compute node instance type in a machine pool. The number and types of nodes available to you depend on your {product-title} subscription.
 ====
 
+. Choose your preference for the Instance Metadata Service (IMDS) type, either using both IMDSv1 and IMDSv2 types or requiring your EC2 instances to use only IMDSv2. The two ways to access instance metadata from a running instance are:
```
Comment (adellape) [inline modules/osd-create-cluster-ccs.adoc:169]: IBM SG suggests avoiding continuing sentences into a list from a lead-in. Maybe:
```suggestion
. Choose your preference for the Instance Metadata Service (IMDS) type, either using both IMDSv1 and IMDSv2 types or requiring your EC2 instances to use only IMDSv2. You can access instance metadata from a running instance in two ways:
```
or
```suggestion
. Choose your preference for the Instance Metadata Service (IMDS) type, either using both IMDSv1 and IMDSv2 types or requiring your EC2 instances to use only IMDSv2. There are two methods for accessing instance metadata from a running instance:
```

**Review by adellape — COMMENTED** [review]
@skopacz1 @EricPonvelle Added one thought, but otherwise LGTM 👍

**Pull request merged**